JWPlayer-SDK-iOS-Demo/ObjCViewController.m

```diff
@@ -0,0 +1,46 @@
+//
+//  ObjCViewController.m
+//  JWPlayer-SDK-iOS-Demo
+//
+//  Created by Amitai Blickstein on 2/26/19.
+//  Copyright © 2019 JWPlayer. All rights reserved.
+//
+
+#import "ObjCViewController.h"
+#import "JWPlayer_SDK_iOS_Demo-Swift.h"
+#import <JWPlayer_iOS_SDK/JWPlayerController.h>
+
+@interface ObjCViewController ()
+
+@property (weak, nonatomic) IBOutlet UIView *playerContainerView;
+@property (nonatomic) JWPlayerController *player;
+
+@end
+
+@implementation ObjCViewController
+
+    
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    self.player = [JWPlayerFactory newPlayerWithDelegate:nil];
+}
+    
+- (void)viewDidAppear:(BOOL)animated {
+    [super viewDidAppear:animated];
+    [self layoutPlayerView:self.player.view];
+}
+
+    - (void)viewWillDisappear:(BOOL)animated
+    {
+        [self.player pause];
```

Comment:
remove this override, this app should only setup.

Reply:
✅